supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc

```diff
@@ -512,13 +513,13 @@ There is no functional difference between the first server that was installed an
 [discrete]
 [[intel-64]]
 ==== image:images/yes.png[yes] Intel 64 (noun)
-*Description*: The Intel 64-bit version of the x86 architecture. Use this format when referring to information that is exclusive to Intel processors. Only use in RHEL documentation. 
+*Description*: The Intel 64-bit version of the x86 architecture. Use this format when referring to information that is exclusive to Intel processors. Only use in RHEL documentation.
 
 Cloud providers can use different formats of this term when using architectures. If you are documenting hard code, commands, or outputs, confer with your SME on the correct format for the specific use case.
 
-Example: 
+Example:
 
-_This feature can run on only Intel 64 processors_ 
+_This feature can run on only Intel 64 processors_
```

Comment:
Suggestion: 

`This feature can run only on Intel 64 processors._`

The placement of 'only' reads strangely as is. I think the guidelines about placement of 'only' would only really be an issue if it came before the verb so I think it's better placed before the preposition in this case. 
*Others might disagree.

There's also a missing period at the end of the sentence.

Reply:
@ndeevy "only" is used correctly. "Only" is a modifier that's very easy to misplace. For example, "This feature can only run on Intel 64 processors" might sound better, but it means "the only thing this feature can do is run on Intel 64 processors", which isn't the case. With the current use of "only", the sentence means "The only thing the feature can run on is Intel 64 processors", which is indeed the correct meaning.

Reply:
Got that @mportman12, TY :-] my suggestion was just to move it one space back to before 'on' (not 'only run on', which _would_ be an issue). `This feature can run only on Intel 64 processors.` still means "The only thing the feature can run on is Intel 64 processors" but, I support your point so can stand down my comment. :-]

Reply:
@ndeevy ah got it! Sorry for the confusion.

Reply:
@ndeevy @mportman12 Oh yes "only" is tricky. I had to read your comments twice to understand 😆 but now I got it and will change accordingly.